src/js/getIllustrations.js

```diff
@@ -1,17 +1,16 @@
+import config from '../config/config';
+
 export default function getIllustrations(bookSlug) {
```

Comment:
so thinking about this now, i remember that we had the try catch because we don't have all the illustrations at this moment. so the test might pass but I think that we might have problems with actually having those illustrations. do you have anyway of checking it out?
If not I can try and see

Reply:
you are more familiar with this, Nathan. So I think it would be better if you try to do that.